packages/perseus/src/widgets/__tests__/radio_test.jsx

```diff
@@ -632,3 +842,94 @@ describe("multi-choice question", () => {
         },
     );
 });
+
+describe("randomized question", () => {
+    const apiOptions: APIOptions = {
+        crossOutEnabled: false,
+    };
+    const [question] = multiChoiceQuestionAndAnswer;
+    const radio1Widget = ((question.widgets["radio 1"]: any): RadioWidget);
+    const radioOptions = radio1Widget.options;
+    const randomizeQuestion: PerseusRenderer = {
+        ...question,
+        widgets: {
+            ...question.widgets,
+            "radio 1": ({
+                ...radio1Widget,
+                options: {
+                    ...radioOptions,
+                    choices: [
+                        {content: "$x=-6$", correct: false},
+                        {content: "$x=4$", correct: true},
+                        {content: "$x=7$", correct: false},
+                        {
+                            content: "There is no such input value.",
+                            isNoneOfTheAbove: true,
+                            correct: false,
+                        },
+                    ],
+                    randomize: true,
+                },
+            }: RadioWidget),
+        },
+    };
+
+    beforeEach(() => {
+        jest.spyOn(Dependencies, "getDependencies").mockReturnValue(
+            testDependencies,
+        );
+    });
+
+    it("should snapshot the same after randomization based on the same seed", () => {
+        // Arrange / Act
+        const {container} = renderQuestion(randomizeQuestion, apiOptions);
```

Comment:
Can you provide a comment here describing where the seed comes from?  Looking at these tests I have no idea how you provided the seed. :) I think you mentioned it's the problem number, and if so, it'd be good to set it explicitly as part of the test because that's an important part of test setup.